tests/unit/h/schemas/api/group_test.py

```diff
@@ -40,6 +40,32 @@ def test_it_raises_if_name_too_long(self, schema):
         with pytest.raises(ValidationError, match="name:.*is too long"):
             schema.validate({"name": "o" * (GROUP_NAME_MAX_LENGTH + 1)})
 
+    def test_it_raises_if_name_isnt_a_string(self, schema):
```

Comment:
There are tests for the sad path here, but what about the happy path? If I copy some text with leading/trailing whitespace and paste it into the name field, the parsed result should be trimmed.